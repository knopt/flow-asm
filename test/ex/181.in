7
2
4.906774227933237
4.427685309153546 -5.887374914910934 -2.2181355038650636 4.930381964120777 0.2744875708345056 9.422902181510374 2.631027408851489 
-6.530713096569773 -2.305876814112069 -1.3373076106789945 4.839839910137147 1.6885396645572044 -1.2292613707345978 5.922942678816048 

2.5806593508050835 2.87888044165185 
