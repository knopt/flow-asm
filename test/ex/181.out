-66.0052767461771 59.14318412089058 -16.330768076416838 -61.34418640706706 52.45949775814893 -125.68483511416588 33.16834301383314 
138.117872354611 -7.569728091877379 -32.7380346053513 -59.657650792926646 26.119812370137335 72.73404769313069 -28.150481745010225 
